scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/EquationSolvingAlgorithms.java

```diff
@@ -385,14 +401,11 @@ public boolean isApproximation() {
         //localDerivative has been computed with what is now xn_m1 and xn_m2
         x_n_double = x_n_double - (ff_x_nminus1.in(Duration.MICROSECONDS) / localDerivative);
         x_n = Duration.of((long) x_n_double, Duration.MICROSECONDS);
-        //outside of domain, diverging
-        if (x_n_double < dxLow_long || x_n_double > dxHigh_long) {
-          throw new DivergenceException("Looking for root out of prescribed domain :[" + dxLow + "," + dxHigh + "]",
-                                        new History<>(history));
-        }
```

Comment:
This is wrong, the secant method might go outside of the initial domain to finally come back. This is not sufficient to declare there is divergence.